spec/project-spec.js

```diff
@@ -1064,9 +1064,16 @@ describe('Project', () => {
           for (let event of events) {
             remaining.delete(event.path);
           }
+          clearTimeout(expireTimeoutId);
```

Comment:
Should you move this where it was?

Reply:
This was moved outside the if to prevent the `expire` error (line 1077/1084) from happening at the same time as the `all events were processed` error